modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js

```diff
@@ -0,0 +1,222 @@
+ * distribution rights of the Software.
+ */
+
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayButton from '@clayui/button';
+import ClayCard from '@clayui/card';
+import {useResource} from '@clayui/data-provider';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm, {ClayInput, ClayRadio, ClayRadioGroup} from '@clayui/form';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+
+const AdminTooltip = ({
+	namespace,
+	setRemovePinHandler,
+	setShowTooltip,
+	showTooltip,
+}) => {
+	const [pinLabel, setPinLabel] = useState(showTooltip.details.label);
+	const [linkedValue, setLinkedValue] = useState(
+		showTooltip.details.linked_to_sku
+	);
+	const [sku, setSku] = useState(showTooltip.details.sku);
+	const [quantity, setQuantity] = useState(showTooltip.details.quantity);
+	const [networkStatus, setNetworkStatus] = useState(4);
+
+	const initialLoading = networkStatus === 1;
+	const loadingAd = networkStatus < 4;
+	const error = networkStatus === 5;
+
+	const {resource} = useResource({
+		fetchPolicy: 'cache-first',
+		link: 'https://rickandmortyapi.com/api/character/',
+		onNetworkStatusChange: setNetworkStatus,
+		variables: {
+			name: sku,
+		},
+	});
+
+	const cardStyle = {
+		left: showTooltip.details.cx,
+		top: showTooltip.details.cy,
+	};
+
+	return (
+		<ClayCard className="admin-tooltip" style={cardStyle}>
+			<ClayCard.Body className="row">
+				<ClayForm.Group className="col-12 form-group-sm text-left">
+					<label htmlFor={`${namespace}pin-position`}>
+						{Liferay.Language.get('position')}
+					</label>
+
+					<ClayInput
+						id={`${namespace}pin-position`}
+						onChange={(event) => setPinLabel(event.target.value)}
+						placeholder={Liferay.Language.get(
+							'insert-your-name-here'
+						)}
+						type="text"
+						value={pinLabel}
+					/>
+				</ClayForm.Group>
+
+				<ClayForm.Group className="col-12 form-group-sm">
+					<ClayRadioGroup
+						className="d-flex justify-content-start mt-4"
+						inline
+						onSelectedValueChange={(val) => setLinkedValue(val)}
+						selectedValue={linkedValue}
+					>
+						<ClayRadio label="Linked to SKU" value="sku" />
+
+						<ClayRadio label="Linked to Diagram" value="diagram" />
+					</ClayRadioGroup>
+				</ClayForm.Group>
+
+				<ClayForm.Group className="col-9 form-group-sm text-left">
+					<label htmlFor={`${namespace}pin-sku`}>
+						{Liferay.Language.get('select-sku')}
+					</label>
+					<ClayAutocomplete>
+						<ClayAutocomplete.Input
+							id={`${namespace}pin-sku`}
+							onChange={(event) => setSku(event.target.value)}
+							placeholder={Liferay.Language.get('type-sku-here')}
+							value={sku}
+						/>
+						<ClayAutocomplete.DropDown
+							active={(!!resource && !!sku) || initialLoading}
+						>
+							<ClayDropDown.ItemList>
+								{(error || (resource && resource.error)) && (
+									<ClayDropDown.Item className="disabled">
```

Comment:
You should use the `disabled` prop on this component, [see the API](https://clayui.com/docs/components/drop-down/api.html)